ansible/roles/couchdb/tasks/deploy.yml

```diff
@@ -88,6 +92,45 @@
     force_basic_auth: yes
   when: (create_users_db.status == 404) and (inventory_hostname == coordinator) and (couchdb.version is version_compare('2.0','>='))
 
+- name: check whether couchdb is clustered
+  uri:
+    url: "{{ db.protocol }}://{{ ansible_host }}:{{ db.port }}/_cluster_setup"
+    method: GET
+    status_code: 200
```

Comment:
can this error and return 40x?

Reply:
In case the cluster is not formed yet, it will return like this:

```
$ curl localhost:5984/_cluster_setup -v
*   Trying ::1...
* Connected to localhost (::1) port 5984 (#0)
> GET /_cluster_setup HTTP/1.1
> Host: localhost:5984
> User-Agent: curl/7.43.0
> Accept: */*
>
< HTTP/1.1 200 OK
< X-CouchDB-Body-Time: 0
< X-Couch-Request-ID: 30fa78e069
< Server: CouchDB/2.1.2 (Erlang OTP/17)
< Date: Sun, 17 Feb 2019 13:59:41 GMT
< Content-Type: application/json
< Content-Length: 29
< Cache-Control: must-revalidate
<
{"state":"cluster_disabled"}
```

Reply:
Surely, it will return `401 Unauthorized` in case the provided credentials are incorrect.